src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -462,8 +462,10 @@
             "auxiliaryRepository": {
                 "error": "There is a problem with the auxiliary repository.",
                 "addAuxiliaryRepository": "Add Auxiliary Repository",
+                "usageDescription": "You can use auxiliary repositories to provide additional Code that students cannot see or modify. The additional Code is inserted at the specified Checkout Directory during the build of the submission.",
```

Comment:
Can you double-check the capitalization here? (esp. `Code`)

Reply:
I thought a bit about it, but stumbled across it aswell - it does not seem to be fully consistent, but `code` is probably the better way.